test/new_users.spec.js

```diff
@@ -0,0 +1,84 @@
+const supertest = require('supertest');
+const app = require('../app');
+
+const request = supertest(app);
+
+describe('If POST /users', () => {
+  it('with complete data, receive an empty response with status 204', async done => {
+    await request
```

Comment:
Ok, this focus it's great! Generally, we create a describe for each case, and within create one it by each thing to check. For example, in this case, the describe could be `with complete data`, and within ,you can create a beforeAll with the request and the response, and two its like 'receive the status code 204' and 'receive an empty response'.
In the first it the expect could be like  `expect(response.statusCode).toBe(204)` and the second `expect(response.body).toBeNull()` or `expect(response.body).toBeUndefined()` (I don't remember exactly the empty body response).
Do you understand? It's because of the test granularity. Is cleaner to understand the test.

Reply:
Also, you must check everything as you can, like empty body, status code, response, the db (yes, you can call a model and check if the user was created). When you create a resource (in this case, users) generally we send 201 (http code for created). Could you refactor the test?